aws/acm.go

```diff
@@ -49,14 +51,24 @@ func getACMCertificateSummaries(api acmiface.ACMAPI) ([]*acm.CertificateSummary,
 }
 
 func getCertificateSummaryFromACM(api acmiface.ACMAPI, arn *string) (*certs.CertificateSummary, error) {
-	params := &acm.DescribeCertificateInput{CertificateArn: arn}
-	resp, err := api.DescribeCertificate(params)
+	params := &acm.GetCertificateInput{CertificateArn: arn}
+	resp, err := api.GetCertificate(params)
```

Comment:
Ca nyou update https://github.com/zalando-incubator/kube-ingress-aws-controller/blob/master/deploy/requirements.md , we do not have the GetCertificate allowed and please add a readme entry how to switch the version, because the iam role needs to be changed, so we also have to increase the version number as before.